docs/operate/reference/components/sensor/lorawan.md

```diff
@@ -0,0 +1,519 @@
+| `dev_addr` | string | Optional | 32-bit hexadecimal **device address** used to identify this device in uplink messages. Required for ABP activation protocol. Found in the device datasheet or in device packaging. |
+| `app_s_key` | string | Optional | 128-bit hexadecimal **application session key** used to decrypt uplink messages. Required for ABP activation protocol. Found in the device datasheet or in device packaging. <br> On `milesight-ct101` and `milesight-em310-tilt`, defaults to `5572404C696E6B4C6F52613230313823`. |
+| `network_s_key` | string | Optional | 128-bit hexadecimal **network session key** used to decrypt uplink messages. Required for ABP activation protocol. Found in the device datasheet or in device packaging. <br> On `milesight-ct101` and `milesight-em310-tilt`, defaults to `5572404C696E6B4C6F52613230313823`. |
+| `gateways` | string[] | **Required** | Name of the [gateway component](#add-a-gateway) in your Viam configuration. |
+| `uplink_interval_mins` | decimal | **Required** | Interval between uplink messages sent from the node, in minutes. Found in the device datasheet, but can be modified. Configured by downlink after initial connection. <br> On `milesight-ct101`, defaults to `10`. On `milesight-em310-tilt`, defaults to `1080`. |
+| `decoder_path` | string | **Required** | Path to a Javascript **decoder script** used to interpret data transmitted from the node. You can use a local path on your device or an HTTP(S) URL that points to a file on a remote server. If the decoder script provides multiple implementations, uses the Chirpstack version. Not compatible with The Things Network decoders. <br> On supported Milesight and Dragino models, defaults to the latest decoder published by the manufacturer on GitHub. |
+| `fport` | string | Optional | 8-bit hexadecimal **frame port** used to send downlinks to the device. Found in the device datasheet. <br> On supported Milesight models, defaults to `"55"` (equivalent to `0x55`). <br> On supported Dragino models, defaults to `"01"` (equivalent to `0x01`). |
+
+{{% alert title="Info" color="info" %}}
+
+Device-specific models, including those for Milesight and Dragino sensors, provide default values for `app_key`, `network_s_key`, `app_s_key`, `fport`, `decoder_path`, and `uplink_interval_mins`.
+
+{{% /alert %}}
+
+### Activation protocols
+
+LoRaWAN networks can use any of the following protocols for communication:
+
+<!-- prettier-ignore -->
+| Name | Value | Key | Session Key | Configuration |
+| ---- | ----- | --- | ----------- | ------------- |
+| Over-The-Air Activation | `OTAA` | Dynamic, generated at join time | Automatically rotated | <ul><li> `dev_eui` </li><li> `app_eui` </li><li> `app_key` </li></ul> |
```

Comment:
```suggestion
| Over-the-air activation | `OTAA` | Dynamic, generated at join time | Automatically rotated | <ul><li> `dev_eui` </li><li> `app_eui` </li><li> `app_key` </li></ul> |
```
[nit]

Reply:
I'm going to ignore this suggestion because this is an acronym and I'm using the [official capitalization](https://www.thethingsnetwork.org/docs/lorawan/end-device-activation/)!